benchmark/worker/messageport.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const common = require('../common.js');
+const { MessageChannel } = require('worker_threads');
+const bench = common.createBenchmark(main, {
```

Comment:
I think this is missing a test in `test/benchmark`?

Reply:
The benchmark parameters are a subset of those in `echo.js`, so I don’t think so?

Reply:
Oh, yes, sorry. Kind of forgot that our benchmark test are for a benchmark group in general.